dotenv.cabal

```diff
@@ -119,10 +119,10 @@ test-suite dotenv-test
 
   build-depends:       base >= 4.9 && < 5.0
                      , dotenv
-                     , megaparsec
-                     , hspec
-                     , process
-                     , text
+                     , megaparsec >= 9.2.2 && <= 9.4.1
+                     , hspec >= 2.9.7 && <= 2.11.4
+                     , process >= 1.6.16 && < 1.7
+                     , text >= 1.2.5 && < 1.3
```

Comment:
The [comment above](https://github.com/stackbuilders/dotenv-hs/pull/176/files#r1312448517) also applies here.